crates/process-event/Cargo.toml

```diff
@@ -11,3 +11,4 @@ reqwest = { version = "0.11.0", features = ["blocking", "json", "multipart"] }
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
 structopt = "0.3.21"
+symbolic-common = { git = "https://github.com/getsentry/symbolic", branch = "fix/demangle-fixes" }
```

Comment:
All changes from this branch are already merged on master from what I see

Reply:
Not quite. We are carrying that branch around for a way too long time already.

We can’t get rid of it quite yet, as that would regress some of our C++ demangling :-( But maybe we can switch from a symbolic branch to maybe a `Cargo.toml` `[patch]` section so we only pull in a specific commit/fix from `cpp_demangle`, and not a whole `symbolic` branch.